src/modules/px4iofirmware/registers.c

```diff
@@ -385,6 +393,32 @@ registers_set(uint8_t page, uint8_t offset, const uint16_t *values, unsigned num
 
 		break;
 
+	case PX4IO_PAGE_CONTROL_TRIM_PWM:
+
+		/* copy channel data */
+		while ((offset < PX4IO_SERVO_COUNT) && (num_values > 0)) {
+
+//			if (*values == 0) {
```

Comment:
We should not commit commented code which has not explaining comments around it.